gauntlet/packages/gauntlet-solana-contracts/src/core/protoSchemas.ts

```diff
@@ -0,0 +1,125 @@
+const header = {
+  // state: {
+  //   type: 'protoimpl.MessageState',
+  // },
+  // sizeCache: {
+  //   type: 'protoimpl.SizeCache',
+  // },
+  // unknownFields: {
+  //   type: 'protoimpl.UnknownFields',
+  // },
+}
+export const offchainDescriptor = {
```

Comment:
Duplicated proto files under monitoring branch here: https://github.com/smartcontractkit/chainlink-solana/pull/10/files#diff-d53509b8d5e82cb783a72db0991c03b36bcdb2c02d4da53188c31523d67e3615R1

Ideally, sync with @topliceanu and agree on a single source for the protobuf declrations.